### ISSUE TITLE
aria-orientation should be allowed on radiogroup

### ISSUE THREAD

**Issue by jnurthen** [dequelabs/axe-core#2145]
Expectation: 
The ARIA spec allows aria-orientation on a radiogroup. aXe should allow this.

Actual: 
aXe allows aria-orientation on all other allowed roles except for radiogroup

Motivation: 
The ARIA spec allows aria-orientation on radiogroup. aXe should not give the message 
`ARIA attribute is not allowed: aria-orientation="vertical"`
aXe should allow this attribute - while it doesn't often do anything useful on a radiogroup if not supported by a browser/AT combination it being there does no harm.
If aXe decides to not allow this attribute for some reason then the message should be clear about why and it should not state that it is not allowed (because it is allowed)

<pre><code>
axe-core version: 3.5.1
</code></pre>

**Comment by WilcoFiers** [comment]
Definitely, go for it. Thanks @jnurthen.

**Comment by straker** [comment]
@jnurthen any update on this?

**Comment by straker** [comment]
This should now be fixed in the latest develop of axe-core.

**Comment by padmavemulapati** [comment]
tested with test file `<h2 id="label_fruit"> Available Fruit </h2>`
`<ul role="listbox" aria-labelledby="label_fruit"  aria-orientation="vertical">`
... its not giving failures
![image](https://user-images.githubusercontent.com/47409579/89427174-9380b100-d758-11ea-95e4-5ab5023364e4.png)

**Comment by jeankaplansky** [comment]
No user-facing docs required.
